include/ada/checkers.h

```diff
@@ -93,6 +93,13 @@ namespace ada::checkers {
    */
   ada_really_inline constexpr uint8_t path_signature(std::string_view input) noexcept;
 
+
+  /** 
+  * A domain cannot be more than 255 octets and have more than 127 labels, each label must have
+  * a maximum of 63 octets.
```

Comment:
Can you add a reference to the specification using `@see`. You might give a context why this is the case and mention:

1. URL spec calls ToASCII with be_strict true
2. ToASCII passes `VerifyDnsLength` if `be_strict` is true.
3. `VerifyDnsLength` checks 253 and 63 limitations (referencing: https://www.unicode.org/reports/tr46/#ToASCII)

Reply:
Let's be as open as possible in here, to avoid communication issues.

Reply:
Thanks for the link!
I was considering:

```
labels          63 octets or less

names           255 octets or less
```
in  https://www.rfc-editor.org/rfc/rfc1035